src/oss/confluent.sh

```diff
@@ -1308,6 +1310,50 @@ extract_json_config() {
     _retval="${parsed_json}"
 }
 
+produce_command() {
+    local topicname="${1}"
+    local args="$@"
+
+    if [[ "x${topicname}" == "x" ]]; then
+        echo "Missing required topic name argument in '${command_name} produce <topicname>'"
+        produce_usage 1
+    fi
+
+    local brokerlist=""
+    if [[ ! "$args" =~ "broker-list" ]]; then
```

Comment:
I don't think we should allow arbitrary `broker-list`. I'd be more in favor to pin it to localhost as with the other services and use `"localhost:${kafka_port}"` (we should extract `kafka_port` before running this command as in `config_connect`)